phpunit.xml.dist

```diff
@@ -10,6 +10,9 @@
          stopOnFailure="false"
          syntaxCheck="false"
          bootstrap="vendor/autoload.php">
+    <php>
+        <ini name="error_reporting" value="E_ALL" />
+    </php>
```

Comment:
testsuites/filters has one line between, same for filters/logging, don't be inconsistent